### PR TITLE
Creature.Self -> Card.Self for Myrkul ETBs

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

**Review by Agetian — DISMISSED** [review]

**Comment by Agetian** [comment]
Looks good and reasonable :)

File: forge-gui/res/cardsfolder/f/flayer_of_the_hatebound.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ Types:Creature Devil
 PT:4/2
 K:Undying
 T:Mode$ ChangesZone | Origin$ Graveyard | Destination$ Battlefield | TriggerZones$ Battlefield | ValidCard$ Creature.YouOwn+Other | Execute$ ReanimateDmg | TriggerDescription$ Whenever CARDNAME or another creature enters the battlefield from your graveyard, that creature deals damage equal to its power to any target.
-T:Mode$ ChangesZone | Origin$ Graveyard | Destination$ Battlefield | ValidCard$ Creature.Self | Secondary$ True | Execute$ ReanimateDmg | TriggerDescription$ Whenever CARDNAME or another creature enters the battlefield from your graveyard, that creature deals damage equal to its power to any target.
+T:Mode$ ChangesZone | Origin$ Graveyard | Destination$ Battlefield | ValidCard$ Card.Self | Secondary$ True | Execute$ ReanimateDmg | TriggerDescription$ Whenever CARDNAME or another creature enters the battlefield from your graveyard, that creature deals damage equal to its power to any target.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/f/flayer_of_the_hatebound.txt:7]: combine them into `Card.Self,Creature.YouOwn+Other`

**Comment by Hanmac** [comment]
Also:
`Breathless Knight` => `Card.Self,Creature.YouCtrl+Other`

**Comment by Northmoc** [comment]
> Also: `Breathless Knight` => `Card.Self,Creature.YouCtrl+Other`

I was wondering about this one

**Review by Northmoc — APPROVED** [review]
Great work; thank you

**Pull request merged**